packages/rps/package.json

```diff
@@ -34,7 +36,7 @@
     "precommit": "lint-staged --quiet"
   },
   "lint-staged": {
-    "{src,tests}/**/*.{ts,tsx}": "yarn lint:check"
+    "{src,tests}/**/*.{ts,tsx}": "eslint"
```

Comment:
Also I recommend making `prettier` a `warn` and then doing `eslint --max-warnings=0`. I find it nice to separate linting issues versus build issues with yellow and red squiggly lines respectively in my editor.